index.ts

```diff
@@ -329,6 +330,43 @@ class Generator {
     await fs.writeFile('./_site/index.html', index, { encoding: 'utf-8' });
   }
 
+  public async generateRSS(): Promise<void> {
+    const feed = new Feed({
+      title: blogTitle,
+      description: blogTitle,
+      id: blogDomain,
+      link: blogDomain,
+      copyright: "All rights reserved 2023, Rémi Berson",
+      feedLinks: {
+        json: `${blogDomain}/json`,
+        atom: `${blogDomain}/atom`
+      },
+      author: {
+        name: "Rémi Berson",
+        link: blogDomain
+      }
+    });
+
+    const posts = Array.from(this.posts.values()).sort(
+      (p1: Post, p2: Post) => p2.date.getTime() - p1.date.getTime(),
+    );
+    
+    posts.forEach(post => {
+      feed.addItem({
+        title: post.title,
+        id: `${blogDomain}/${post.url}`,
+        link: `${blogDomain}/${post.url}`,
+        content: post.content,
```

Comment:
Nit. Should content be the raw Markdown or HTML?

From what I could read in their source, I think it might be expected to be `HTML` (in which case we could use `post.html` maybe? Although I'm not 100% sure if they need the whole thing with `<html><head>...` or just the body?)

Reply:
Seems like there isn't a consensus but many seem to use the html body like you suggested.